app/src/components/Content/Content.styles.js

```diff
@@ -2,7 +2,7 @@ import styled from "styled-components";
 
 export const StyledSection = styled.section`
   max-width: 80rem;
-  height: 100vh;
+  height: 110vh;
```

Comment:
I don't think that this is correct. I've never had to use more than 100vh. Please try to find a better solution

Reply:
I changed it to 100vh and added 50px of margin-bottom.